src/Bridges/NetteDI/MigrationsExtension.php

```diff
@@ -22,6 +22,7 @@ class MigrationsExtension extends Nette\DI\CompilerExtension
 		'phpParams' => [],
 		'driver' => NULL,
 		'dbal' => NULL,
+		'diffGenerator' => NULL, // false|doctrine
```

Comment:
kdyby to byl JS, tak bych místo NULL použil undefined a místo false null; teď null = autodetect a false = disable autodetection;

možná bude trochu lepší použít místo null true? tj. pattern co dg občas používá